schemas/iso19139.che/src/main/plugin/iso19139.che/index-fields/default.xsl

```diff
@@ -429,7 +429,17 @@
             </xsl:for-each>
         </xsl:for-each>
 
-
+        <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+        <!-- === maintenance Info (AAP) === -->
+        
+        <xsl:choose>
+          <xsl:when test="gmd:identificationInfo/che:CHE_MD_DataIdentification/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:keyword/gco:CharacterString[text() = 'AAP-Bund']">
+                <Field name="AAP" string="true" store="true" index="true"/>
```

Comment:
Live-testing on dev, it seems that the modification in default-language.xsl is actually necessary.